examples/viewer.html

```diff
@@ -539,13 +559,71 @@
       return s;
     }
 
+    /**
+     * Parses the query string of an URL. This method returns a simple key/value
+     * map. If there are duplicate keys the latest value is returned.
+     * @param {string} queryString
+     * @return {!Object<string>}
+     */
+    function parseQueryString(queryString) {
```

Comment:
I took these from the `pwa.js` since it also accepts an `href` from a hash param `#href=http://blah`